flask/app.py

```diff
@@ -827,7 +829,7 @@ def run(self, host=None, port=None, debug=None, **options):
         """
         from werkzeug.serving import run_simple
         if host is None:
-            host = '127.0.0.1'
+            host = 'localhost'
```

Comment:
This is not equivalent to `127.0.0.1` - using `localhost` may result in `::1` being used instead isince `localhost` has both IPv4 and IPv6 addresses